autogen/src/gen_gql_schema.rs

```diff
@@ -684,6 +693,7 @@ fn generate_rc_file(file: File) -> Result<()> {
     let mut written_directives = HashSet::new();
 
     let mut extra_it = BTreeMap::new();
+    extra_it.insert("KeyValue".to_string(), ExtraTypes::KeyValue);
```

Comment:
we can not insert it directly here

Reply:
updated, please check

Reply:
Please do not mark unresolved stuff as resolved

Reply:
Why we can't insert it directly here?

Reply:
updated, please check @ssddOnTop